### PR TITLE
WIP faction mission building: bioshop

### PR DESCRIPTION
**Pull request by faefux**
## Summary

```SUMMARY: Content "Mission building for WIP Faction; Outsiders."```

#### Purpose of change
Starting to add this WIP faction a little at a time.

#### Describe the solution
Going to upload the maps first and then start uploading some stripped d

#### Describe alternatives you've considered
rm -rf'ing this whole faction because i feel really self-conscious about it.

#### Additional context
Context behind this: This is a mission to just go and kill a zombie_bio_op and a zombie_nullfield in the North-Eastern treatment room.   There will also be a ledger that spawns in the office safe that isn't the focus of the mission but will lead to a mission  tree for Kristina if you give it to her. 

This should be a hard encounter and the loot reflects this as well the fact that it's supposed to have been untouched since the cataclysm.

### CODE REVIEW

**Comment by faefux** [comment]
The error with travis is specials.json, i linted what i have but the damn thing isn't letting me edit specials.json on the web app.  Probably the file is to big and issues with browser in that regard.  I'll try to deal with this tomorrow and if not i'll close and just upload all the faction stuff as a separate mod or something.  Sorry for the time wasting if I can't get it done.. :(

**Comment by anothersimulacrum** [comment]
I'd suggest you use an application such as GitHub desktop or the git CLI, the web interface is pretty awful.

**Comment by faefux** [comment]
> I'd suggest you use an application such as GitHub desktop or the git CLI, the web interface is pretty awful.

I can't right now.  But i also don't want to never upload the ton of content i'm writing and have written because of an absolute bollox web app.  :(

**Comment by Petethegoat** [comment]
> There will also be a ledger that spawns in the office safe that isn't the focus of the mission but will lead to a mission tree for Kristina if you give it to her.

This sounds really cool. I love things like this where it's not explicitly signposted what to do.

**Comment by faefux** [comment]
> > There will also be a ledger that spawns in the office safe that isn't the focus of the mission but will lead to a mission tree for Kristina if you give it to her.
> 
> This sounds really cool. I love things like this where it's not explicitly signposted what to do.

There's lots of this.  I tried to add lots of conditional conversation stuff that wasn't clear what triggered it and things like this, like for example i bucked the trend of putting [INT[ infront of an int based conversation option which seems to be the trend atm but very much something i don't think is a great idea in the context of an RPG and dialogue.   This will be far more interesting if we will ever get checks against skills rather than just attributes..  

When i've been writing dialogue and missions I've been thinking of fallout 2 and the planescape torment video game rather than the modern 'gamey' games like fallout  3 or skyrim when it comes to NPC interaction and missions..  There are also missions where the NPC gives you the wrong name of things or sends you to the wrong place simply because they remembered wrong or didn't know the directions..  Most of this is just through conversation effects, etc.  I wish I could add missions like 'go to police station' or 'kill the bandits' and not have the mission point straight to the location on the map (maybe there is a way to do this. I'd appreciate it if someone could let me know if  so!) so I could add more of this kind of thing.

File: data/json/overmap/overmap_special/specials.json [hunk 1]
```diff
@@ -5434,5 +5434,15 @@
...
+  {
+    "type": "overmap_special",
+    "id": "underground_bionics_clinic",
+    "overmaps": [ { "point": [ 0, 0, 0 ], "overmap": "bioshop_house" }, { "point": [ 0, 0, -1 ], "overmap": "bioshop_basement" } ],
+    "locations": [ "land" ],
+    "city_distance": [ 0, 1 ],
+    "city_sizes": [ 0, 12 ],
+    "occurrences": [ 0, 0 ],
+    "rotate": false
+  }
```
Comment (AMurkin) [inline data/json/overmap/overmap_special/specials.json]:
```suggestion
  }
```

**Comment by ZhilkinSerg** [comment]
This issue has been closed due to lack of activity. This does not mean that we do not value the issue. Feel free to request that it be re-opened if you are going to actively work on it

**Pull request closed without merging**